src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -463,3 +463,103 @@
 		color: #405bfb;
 	}
 }
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu {
+	padding-top: var(--tec-spacer-6);
+
+	a {
```

Comment:
Normally, it'd be recommented in BEM that each element has its class. So, this `a` could be `.tec-tickets__admin-settings-tickets-commerce-section-menu-link` and then use the class over the `a` element in the HTML.